### PR TITLE
LVJAV-18

### PR DESCRIPTION
**Pull request by oprokipchuk**

### CODE REVIEW

File: src/main/java/com/softserve/ita/java442/cityDonut/model/Media.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+@Builder
+public class Media {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Column(name = "name", length = 30, nullable = false)
+    private String name;
+
```
Comment (martavoloshyn) [inline src/main/java/com/softserve/ita/java442/cityDonut/model/Media.java:19]: add @JoinColumn to all ManyToOne

File: src/main/java/com/softserve/ita/java442/cityDonut/model/Media.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
+package com.softserve.ita.java442.cityDonut.model;
+
+import lombok.Builder;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "media",
+        indexes = {@Index(name = "media_name_index", columnList = "name", unique = true)})
+@Builder
```
Comment (NazarRyvak) [inline src/main/java/com/softserve/ita/java442/cityDonut/model/Media.java:10]: add @AllArgsConstructor and @NoArgsConstructor

File: src/main/java/com/softserve/ita/java442/cityDonut/model/Media.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
+package com.softserve.ita.java442.cityDonut.model;
+
+import lombok.Builder;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "media",
+        indexes = {@Index(name = "media_name_index", columnList = "name", unique = true)})
```
Comment (NazarRyvak) [inline src/main/java/com/softserve/ita/java442/cityDonut/model/Media.java]: move columnList, unique to @Column

File: src/main/java/com/softserve/ita/java442/cityDonut/model/Media.java [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+@Builder
+public class Media {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
```
Comment (NazarRyvak) [inline src/main/java/com/softserve/ita/java442/cityDonut/model/Media.java:14]: change GenerationType...needs Identity

**Pull request closed without merging**